docs/usage_guide/optimization_guide/index.rst

```diff
@@ -0,0 +1,230 @@
+Optimization Guide
+==================
+
+Here you can find more information about Modin's optimizations both for a pipeline of operations as well as for specific operations.
```

Comment:
I would like to add a comment here at the beginning about how Modin will do most of the optimization for you.
```suggestion
Modin has chosen default values for a lot of the configurations here that provide excellent performance in most 
cases. This page is for those who love to optimize their code and those who are curious about existing optimizations
within Modin. Here you can find more information about Modin's optimizations both for a pipeline of operations as 
well as for specific operations.
```